app/api/enrollments.js

```diff
@@ -0,0 +1,123 @@
+const router = require('express').Router({ mergeParams: true })
+const db = require('../models/index')
+const courseService = require('../services/course_service')
+const enrollmentService = require('../services/enrollment_service')
+const sectionService = require('../services/section_service')
+const { requireAuthentication } = require('../../lib/auth')
+const string_helpers = require('../../lib/string_helpers')
+const { logger } = require('../../lib/logger')
+
+// GET /courses/:course_id/enrollments
+// endpoint handles teacher getting a roster for a given course
+router.get('/', requireAuthentication, async function (req, res) {
```

Comment:
I wonder if this just needs to be a separate api call, but my concern for the frontend side of things is how I'm going to get the role of my user for different courses, so I know what elements are needed.  Since only the teacher can use this call, maybe we'll need one eventually that's tailor fit for the roles. We can talk more Wednesday evening.